### PR TITLE
doc: Add a test for the longhorn manager starting and upgrade scalability issue

### PR DESCRIPTION
**Pull request by shuo-wu**
Longhorn/longhorn#3087

### CODE REVIEW

**Review by innobead — APPROVED** [review]
LGTM

File: docs/content/manual/release-specific/v1.3.0/test-longhorn-manager-starting-scalability.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+---
+title: Test longhorn manager pod starting scalability
+---
+
+### Test step
+1. Deploy a cluster with multiple nodes. e.g., 20 worker nodes.
+2. Launch an old Longhorn version without the fix PR. e.g., Longhorn version v1.2.3.
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.3.0/test-longhorn-manager-starting-scalability.md:7]: Nit: It's better to provide example as v1.2.2 or v1.1.2 as in step 6, user will need to upgrade to another version which  doesn't have fix.

**Review by khushboo-rancher — APPROVED** [review]
LGTM, just a nit.

File: docs/content/manual/release-specific/v1.3.0/test-longhorn-manager-starting-scalability.md (hunk 1 above)
Comment (shuo-wu) [reply docs/content/manual/release-specific/v1.3.0/test-longhorn-manager-starting-scalability.md:7]: You may need to build a test image based on the version you used in step 2 by yourself. 

And I will add this statement in the doc.

**Comment by innobead** [comment]
cc @longhorn/qa all QAs need to pay attention on this.

**Pull request merged**